pkgs/servers/irrd/default.nix

```diff
@@ -0,0 +1,157 @@
+{ lib
+, python3
+, fetchPypi
+, git
+, postgresql
+, postgresqlTestHook
+, redis
+}:
+
+let
+  py = python3.override {
+    packageOverrides = self: super: {
```

Comment:
Afaicr, the convention for overlayish things is `final: prev: `

But no strong feelings.

Reply:
fixed